Makefile

```diff
@@ -233,15 +233,10 @@ lvm-driver-image: lvm-driver
 	cd buildscripts/${CSI_DRIVER} && sudo docker build -t ${IMAGE_ORG}/${CSI_DRIVER}:${IMAGE_TAG} ${DBUILD_ARGS} . && sudo docker tag ${IMAGE_ORG}/${CSI_DRIVER}:${IMAGE_TAG} quay.io/${IMAGE_ORG}/${CSI_DRIVER}:${IMAGE_TAG}
 	@rm buildscripts/${CSI_DRIVER}/${CSI_DRIVER}
 
-.PHONY: ci-setup
-ci-setup:
+.PHONY: ci
+ci:
 	@echo "--> Running ci test";
-	$(PWD)/ci/setup.sh
```

Comment:
Is the setup now done [here](https://github.com/openebs/lvm-localpv/pull/26/files?file-filters%5B%5D=.docs&file-filters%5B%5D=.go&file-filters%5B%5D=.md&file-filters%5B%5D=.mod&file-filters%5B%5D=.proto&file-filters%5B%5D=.sh&file-filters%5B%5D=.sum&file-filters%5B%5D=.txt&file-filters%5B%5D=.yaml&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile#diff-6c64843b28c3d98c5ceeee76870521cf40ee8d74de3c3d5ac133cf947977f7ebR19)

Reply:
there was no setup.sh earlier. everything is done in ci-test.sh file.